### DISCUSSION TITLE
Don't have an university email hence unable to enroll in this program

### DISCUSSION THREAD

**Discussion by IshikaHere** [community/community#63100] [GitHub Education]
### Hi everyone,

 I applied for it once I got accepted but a few days later it's showing me that I have to renew it. I want to renew it but having some issues with my **_student email_**. **Our university doesn't provide student email IDs** to **students**. I am not able to process my application any further because I don't have an email ID. I can give **any other type of proof** that I am a student at this university but I can't provide it since I don't have my email id.

**Comment by humairshoukat** [comment]
If you don't have email id you can submit your Student ID Card with clear expiration date mentioned on it.

**Reply by IshikaHere** [reply]
where should i send it ?

**Comment by samus-aran** [comment]
Hi @IshikaHere 👋 

As you can imagine there are many types of academic verification and our system is constantly evolving to accommodate this without setting the bar too high. However sometimes we may require a little more evidence from the student applying and we thank you for being patient. In this case it would be the Academic email(s) which has been already successfully associated with this school.
We would recommend to you speak to your institution regarding to acquiring an email address to proceed.

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

Hope this helps! Let us know how you get along 👍
